data/remnant/remnant missions.txt

```diff
@@ -940,6 +940,143 @@ mission "Remnant: Learn Sign Follow Up"
+			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?"`
+			`	He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`
+			branch confirmation
+				has "deep: did not reveal remnant"
+			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
+			goto investigatechoice
+			label confirmation
+			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system and now know that quantum keystones can stabilize it. He quickly picks up on the potential threat that this development represents.`
+			`	Thinking carefully, he asks, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
+			choice
+				`	"I'd be happy to."`
+				`	"I'd prefer not to."`
+					decline
+			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
+				accept
+	on complete
+		conversation
+			`Arriving on Bounty, you take a discreet look around the spaceport, but you don't find the researchers or any sign of them. It appears that they packed up and left since the last time you saw them. Time to look around for some info.`
+
+
+
+mission "Remnant: Follow-up on Terminus Researchers 2A"
+	name "Check up on Terminus Researchers"
+	description "Tracking down the wormhole researchers."
+	source Bounty
+	destination
+		government Deep
+	to offer
+		has "Remnant: Follow-up on Terminus Researchers: done"
+	on offer
+		conversation
+			`A few hours of searching later, you find a lead: It seems that launching probes into the Terminus system isn't exactly a common activity, so their permits stand out from the rest. A helpful clerk in the spaceport office pulled up their application and found that they are listed as part of a university in the Deep, although it doesn't mention which planet their specific campus is on. A quick check of the directory lists a number of planets where they could be found.`
```

Comment:
This seems like unnecessary padding to me. Why couldn't I just look up the university's location of I know what it is?